modin/experimental/core/execution/native/implementations/omnisci_on_native/utils.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Utilities for OmniSci on Native execution."""
+
+import sys
+
+
+if sys.platform == "linux":
+    prev = sys.getdlopenflags()
+    sys.setdlopenflags(1 | 256)  # RTLD_LAZY+RTLD_GLOBAL
+
+try:
+    from omniscidbe import PyDbEngine  # noqa
+except ModuleNotFoundError:  # fallback for older omniscidbe4py package naming
+    from dbe import PyDbEngine  # noqa
+
+if sys.platform == "linux":
+    sys.setdlopenflags(prev)
```

Comment:
```suggestion
try:
    from omniscidbe import PyDbEngine  # noqa
except ModuleNotFoundError:  # fallback for older omniscidbe4py package naming
    from dbe import PyDbEngine  # noqa
finally:
    if sys.platform == "linux":
        sys.setdlopenflags(prev)
```
otherwise, we will not restore original flags if neither of the modules installed

Reply:
fixed